tree/ntuple/v7/inc/ROOT/RFieldVisitor.hxx

```diff
@@ -126,19 +140,51 @@ public:
       fAvailableSpaceValueString = fWidth - 6 - fAvailableSpaceKeyString;
    }
 };
-
+// clang-format off
+/**
+\class ROOT::Experimental::RValueVisitor
+\ingroup NTuple
+\brief Traverses through an ntuple to display its entries.
+    
+Each visit outputs the entry of a single field as in a .json file.
+*/
+// clang-format on
+// NOTE (lesimon): currently no support for char, std::array, standard arrays,
```

Comment:
In fact, there is now support for arrays etc, no?

Reply:
Yes, the comment was outdated and now it is deleted. The supported types are all types with a RField specialisation except for VecOps and std::variant.